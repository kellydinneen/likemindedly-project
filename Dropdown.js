import React from 'react';
import { StyleSheet, Text, View, Image } from 'react-native';

const Dropdown = () => {

  return (
    <View style={styles.dropdown}>
      <View style={styles.dropdownGroup}>
        <Image style={styles.icon} source={require('./assets/edit.png')}/>
        <Text style={[{marginLeft: -8}, styles.text]}>Rename</Text>
      </View>
      <View style={styles.dropdownGroup}>
        <Image style={styles.icon} source={require('./assets/copy.png')}/>
        <Text style={[{marginLeft: -8}, styles.text]}>Copy</Text>
      </View>
      <View style={[{
        borderTopColor: '#E8E8E8',
        borderStyle: 'solid',
        borderTopWidth: 1
      }, styles.dropdownGroup]}>
        <Text style={[{color: 'red', marginTop: 10, marginLeft: 5}, styles.text]}>Cancel Event</Text>
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  dropdown: {
    alignSelf: 'flex-end',
    marginTop: 225,
    marginRight: 37,
    paddingTop: 10,
    shadowColor: "black",
    shadowOffset:{
      width: 0,
      height: 3
    },
    shadowOpacity: 0.15,
    shadowRadius: 2,
    height: 110,
    width: 100,
    borderRadius: 3,
    borderTopRightRadius: 0,
    backgroundColor: '#fff',
    alignItems: 'baseline',
    justifyContent: 'center'
  },
  dropdownGroup: {
    margin: 5,
    width: 90,
    flexDirection: 'row',
    justifyContent: 'flex-start',
    alignItems: 'flex-start'
  },
  text: {
    fontSize: 12,
    fontFamily: 'NotoSans_400Regular'
  },
  icon: {
    marginLeft: -10,
    height: 15,
    resizeMode: 'contain'
  }
});

export default Dropdown;
