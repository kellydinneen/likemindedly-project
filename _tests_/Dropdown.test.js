import React from 'react';
import renderer from 'react-test-renderer';
import Dropdown from '../Dropdown';

test('renders correctly', () => {
  const tree = renderer.create(<Dropdown />).toJSON();
  expect(tree).toMatchSnapshot();
});
