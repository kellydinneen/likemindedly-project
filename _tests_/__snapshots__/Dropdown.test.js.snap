// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "baseline",
      "alignSelf": "flex-end",
      "backgroundColor": "#fff",
      "borderRadius": 3,
      "borderTopRightRadius": 0,
      "height": 110,
      "justifyContent": "center",
      "marginRight": 37,
      "marginTop": 225,
      "paddingTop": 10,
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 3,
        "width": 0,
      },
      "shadowOpacity": 0.15,
      "shadowRadius": 2,
      "width": 100,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "margin": 5,
        "width": 90,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/edit.png",
        }
      }
      style={
        Object {
          "height": 15,
          "marginLeft": -10,
          "resizeMode": "contain",
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "marginLeft": -8,
          },
          Object {
            "fontFamily": "NotoSans_400Regular",
            "fontSize": 12,
          },
        ]
      }
    >
      Rename
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "margin": 5,
        "width": 90,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/copy.png",
        }
      }
      style={
        Object {
          "height": 15,
          "marginLeft": -10,
          "resizeMode": "contain",
        }
      }
    />
    <Text
      style={
        Array [
          Object {
            "marginLeft": -8,
          },
          Object {
            "fontFamily": "NotoSans_400Regular",
            "fontSize": 12,
          },
        ]
      }
    >
      Copy
    </Text>
  </View>
  <View
    style={
      Array [
        Object {
          "borderStyle": "solid",
          "borderTopColor": "#E8E8E8",
          "borderTopWidth": 1,
        },
        Object {
          "alignItems": "flex-start",
          "flexDirection": "row",
          "justifyContent": "flex-start",
          "margin": 5,
          "width": 90,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "red",
            "marginLeft": 5,
            "marginTop": 10,
          },
          Object {
            "fontFamily": "NotoSans_400Regular",
            "fontSize": 12,
          },
        ]
      }
    >
      Cancel Event
    </Text>
  </View>
</View>
`;
